zerocopy-derive/src/lib.rs

```diff
@@ -1093,16 +1067,26 @@ enum Trait {
     Sized,
 }
 
+impl ToTokens for Trait {
+    fn to_tokens(&self, tokens: &mut TokenStream) {
+        let ident = Ident::new(&format!("{:?}", self), Span::call_site());
+        tokens.extend(core::iter::once(TokenTree::Ident(ident)));
+    }
+}
+
 impl Trait {
-    fn path(&self) -> Path {
-        let span = Span::call_site();
-        let root = if *self == Self::Sized {
-            quote_spanned!(span=> ::zerocopy::macro_util::core_reexport::marker)
-        } else {
-            quote_spanned!(span=> ::zerocopy)
-        };
-        let ident = Ident::new(&format!("{:?}", self), span);
-        parse_quote_spanned! {span=> #root::#ident}
```

Comment:
Does the new impl discard span information, or am I misreading? If so, is it possible to preserve span information, or is there a reason we should discard it?

Reply:
`quote` will handle the span information correctly in general, I think this should keep the hygiene information correct.